static/app/components/replays/replayContext.tsx

```diff
@@ -324,22 +260,16 @@ export function Provider({
     replayerRef,
   });
 
-  const setReplayFinished = useCallback(() => {
-    setFinishedAtMS(replayerRef.current?.getCurrentTime() ?? -1);
-    setIsPlaying(false);
-  }, []);
-
-  const getCurrentTime = useCallback(
+  const getCurrentPlayerTime = useCallback(
```

Comment:
I renamed `getCurrentTime()` to `getCurrentPlayerTime()` to distinguish between the underlying player time (which isn't visible outside of this context) and the visual time we display (which is `currentTIme` and used by the lower components)